### DISCUSSION TITLE
list of the particles position

### DISCUSSION THREAD

**Discussion by ariel234156** [nextpyp/.github#119] [Q&A] [answered]
Hi,
I choose particles manually. After I run the pre-processing, where can I find the list of positions?
In addition, I want to extract the position list to relion 5. Do you know how I can do it?
Thank you.

**Comment by abartesaghi** [comment, marked as answer]
After picking particles manually, select "manual" as the detection method (Particle detection tab) and select the list of positions from the drop-down menu. Then create a refinement block downstream and run one iteration of refinement selecting Skip refinement (Refinement tab) and with all constrained refinement options unchecked (Constrained refinement tab) and run the block, Once the run is finished, select the option to Export metadata (Export tab) and select the location of the `Input parfile` from the current refinement block (`frealign/maps/*.par.bz2`). Re-running the refinement block will produce `.star` files that you can import directly into Relion. You will be able to see the coordinates in the `*_particles.star` file (columns `_rlnCoordinateX`, `_rlnCoordinateY` and `_rlnCoordinateZ`).

**Reply by ariel234156** [reply]
Thank you,
It works very well in relion 4, 
do you know if there is an easy wat to convert it to relion 5?

**Reply by yezhou867** [reply]
Hi, did you try using  RELION5 Tomo import Coordinates tab only with both particle.star and tomograms.star and leave other importing methods tabs empty (only give it general info). I think that could work properly to get the same tomogram reconstruction and particles coordinates.  Please let me know if you have problems about this.

**Reply by ariel234156** [reply]
@yezhou867  
It works in RELION5 but is just a 3D sub-tomogram. If I tried to extract the particles to 2D stacks as I need in RELION5, I couldn't.
There is an option to extract the coordinates to 2D stacks to relion 5 ?

**Reply by abartesaghi** [reply]
Hi, nextPYP doesn't save particle stacks, so it is not possible to export those to use in other programs right now. The export job will simply produce the necessary metadata (i.e., tilt-geometry, particle locations and alignments) so that they can be imported into relion, but the extraction itself (whether it is sub-tomograms or particle stacks) will need to be done inside relion.
